vsphere/metadata.csv

```diff
@@ -295,6 +295,8 @@ vsphere.host.summary.runtime.connectionState,count,300,,,The host connection sta
 vsphere.host.summary.runtime.inMaintenanceMode,count,300,,,The flag to indicate whether or not the host is in maintenance mode. This flag is set when the host has entered the maintenance mode. It is not set during the entering phase of maintenance mode. Tagged by `inMaintenanceMode`.,0,vsphere,host maintenance mode,
 vsphere.host.summary.runtime.powerState,count,300,,,The host power state. Tagged by `powerState`.,0,vsphere,host power state,
 vsphere.datastore.count,gauge,,,,Timeserie with value 1 for each Datastore. Make 'sum by {X}' queries to count all the Datastores with the tag X.,0,vsphere,datastore count,
+vsphere.datastore.summary.freeSpace,gauge,300,,,"Available space of this datastore, in bytes. The server periodically updates this value. It can be explicitly refreshed with the Refresh operation.",0,vsphere,datastore freespace,
+vsphere.datastore.summary.capacity,gauge,300,,,"Maximum capacity of this datastore, in bytes. This value is updated periodically by the server. It can be explicitly refreshed with the Refresh operation.",0,vsphere,datastore capacity,
```

Comment:
nit - you could add the unit here

```suggestion
vsphere.datastore.summary.freeSpace,gauge,300,byte,,"Available space of this datastore, in bytes. The server periodically updates this value. It can be explicitly refreshed with the Refresh operation.",0,vsphere,datastore freespace,
vsphere.datastore.summary.capacity,gauge,300,byte,,"Maximum capacity of this datastore, in bytes. This value is updated periodically by the server. It can be explicitly refreshed with the Refresh operation.",0,vsphere,datastore capacity,
```

Reply:
good catch!